ui/src/features/Editor/components/BottomPanel/components/Map/index.tsx

```
@@ -1,32 +1,13 @@
-  mapMode: MapMode;
-  setMapMode?: (mode: MapMode) => void;
+  className?: string;
 };
 
-const Map: React.FC<Props> = ({ mapMode, setMapMode }) => {
-  const t = useT();
-  const mapModes: { key: MapMode; value: string }[] = [
-    { key: "2d-map", value: t("2D") },
-    { key: "3d-map", value: t("3D") },
-  ];
+const Map: React.FC<Props> = ({ className }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)